kubernetes-addons/Observability/README.md

```diff
@@ -0,0 +1,137 @@
+# How-To Setup Observability for OPEA Workload in Kubernetes
+
+This guide provides a step-by-step approach to setting up observability for the OPEA workload in a Kubernetes environment. We will cover the setup of Prometheus and Grafana, as well as the collection of metrics for Gaudi hardware, OPEA/chatqna including TGI,TEI-Embedding,TEI-Reranking and other microservies, and PCM.
+
+## 1. Setup Prometheus & Grafana
+
+Setting up Prometheus and Grafana is essential for monitoring and visualizing your workloads. Follow these steps to get started:
+
+### Step 1: Install Prometheus&Grafana
+
+```
+kubectl create ns monitoring
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm install prometheus-stack prometheus-community/kube-prometheus-stack --version 55.5.1 -n monitoring
+```
+
+### Step 2: Verify the installation:
+
+```
+kubectl get pods -n monitoring
+```
+
+### Step 3: Port-forward to access Grafana:
+
+```
+kubectl port-forward service/grafana 3000:80
+```
+
+### Step 4: Access Grafana:
+
+Open your browser and navigate to http://localhost:3000. Use "admin/prom-operator" as the username and the password to login.
+
+## 2. Metric for Gaudi Hardware(v1.16.2)
+
+To monitor Gaudi hardware metrics, you can use the following steps:
+
+### Step 1: Install daemonset
+
+```
+kubectl create -f https://vault.habana.ai/artifactory/gaudi-metric-exporter/yaml/1.16.2/metric-exporter-daemonset.yaml
+```
+
+### Step 2: Install metric-exporter
+
+```
+kubectl create -f https://vault.habana.ai/artifactory/gaudi-metric-exporter/yaml/1.16.2/metric-exporter-service.yaml
```

Comment:
Why these files are included to PR when doc tells to install them from the network?

Reply:
Just in case the user can't not get the yaml files from network

Reply:
I don't think it is a good practise to include extra files to repos which are available online. We need to assume there is network connectivity.

Reply:
OK, I 've already remove the metric-exporter-service.1.16.2.yaml and metric-exporter-daemonset.yaml